### PR TITLE
Fixed Issue no #3426

### LINKED ISSUE #3426: [Bug] Tiny padding between field name text field and field type selector button
### Bug Description

There should be a 12 px vertical whitespace between. The current 4px is awfully cramped.

Screenshot 2023-09-17 at 8 04 26 PM

### How to Reproduce

1. Open a grid
2. Click on the field header of any field
3. Click on edit field
4. Observe that the spacing is tiny

### Expected Behavior

There should be ample spacing between the field name textfield and the field type button

### Operating System

macOS

### AppFlowy Version(s)

0.3.1

### Screenshots

_No response_

### Additional Context

_No response_

### PR DESCRIPTION
**Pull request by aryan-more**
fixes #3426

Added Padding Between Edit Textfield and Select Type Dropdown

- [✅ ] My code adheres to the [AppFlowy Style Guide](https://appflowy.gitbook.io/docs/essential-documentation/contribute-to-appflowy/software-contributions/submitting-code/style-guides)
- [✅ ] I've listed at least one issue that this PR fixes in the description above.
- [❌ ] I've added a test(s) to validate changes in this PR, or this PR only contains semantic changes.
- [✅ ] All existing tests are passing.

### CODE REVIEW

File: frontend/appflowy_flutter/pubspec.lock [hunk 1]
```diff
@@ -238,10 +238,10 @@ packages:
     dependency: "direct main"
     description:
       name: collection
-      sha256: "4a07be6cb69c84d677a6c3096fcf960cc3285a8330b4603e0d463d15d9bd934c"
+      sha256: f092b211a4319e98e5ff58223576de6c2803db36221657b46c82574721240687
```
Comment (LucasXu0) [inline frontend/appflowy_flutter/pubspec.lock:241]: @aryan-more Please don't update the pubspec.lock

**Review by richardshiue — CHANGES_REQUESTED** [review]
A good start! A few suggestions:

- We have a VSpace class that you can use which is a bit more readable.
- The height of 8 and adding the 4 pre-exisiting whitespace to the top and bottom, makes this 16 in total, not 12
- If there aren't buttons to the bottom (see the primary field or the "Name" field in a new grid), the whitespace shouldn't be added.

**Comment by aryan-more** [comment]
@richardshiue Thanks for suggesting the VSpace Widget. I added VSpace only to the Type Selection Dropdown since I noticed that the other two widgets already have padding, which is why I didn't add it to them.

**Comment by aryan-more** [comment]
@richardshiue i was searching where does those two delete and hide button shows up i didn't able to find them ,thanks for sharing screenshot and it's now fixed

**Review by richardshiue — APPROVED** [review]
LGTM

**Comment by richardshiue** [comment]
Merged, thanks for contributing!

**Pull request merged**